modules/axp/Animal/lib/Animal.pm

```diff
@@ -22,76 +26,55 @@ our $VERSION = '0.01';
 
 Don't use this class directly; instead instantiate subclasses of it.
 
-=head1 EXPORT
-
 =head1 SUBROUTINES/METHODS
 
-=head2 named
-
-Construct a new Animal, with the given name
-
-=cut
-
-sub named {
-	ref ( my $class = shift ) and croak "Static constructor used as instance call";
-	my $name = shift or croak "Need to provide a name";
-	my $self = { Name => $name, Colour => $class->default_colour };
-	return bless $self, $class;
-}
```

Comment:
I've done away with the `named` constructor, in favour of the standard Moose `new( name => ... )`